deepdrivemd/aggregation/stream/aggregator.py

```diff
@@ -0,0 +1,209 @@
+import numpy as np
+from deepdrivemd.aggregation.stream.config import StreamAggregation
+from deepdrivemd.utils import Timer, timer, parse_args, intarray2hash
+import glob
+import time
+import os
+import sys
+import adios2
+import math
+import queue
+import subprocess
+import itertools
+
+
+def find_input(cfg: StreamAggregation):
+    """
+    Find and return simulation adios streams.
+    Wait until those are created by simulations.
+    Returns the list of *.sst files associated with the corresponding adios streams.
+    """
+    while True:
+        bpfiles = glob.glob(str(cfg.experiment_directory) + "/*/*/*/md.bp*")
```

Comment:
Python Path objects have a glob function. Can update to: 
`bpfiles = list(cfg.experiment_directory.glob("/*/*/*/md.bp*"))`

It returns a generator, so that is why the cast to a list is needed to have the same output as glob.glob. If you just need an iterable, then it doesn't need the list wrapper.

Reply:
Done. However, using Path instead of str just makes things more complicated without any real benefit.

Reply:
Pathlib has really nice utilities, but we are not using them in the streaming version.